tests/impl.cpp

```diff
@@ -2568,12 +2568,33 @@ result_t test_mm_storeu_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_storeu_si16(const SSE2NEONTestImpl &impl, uint32_t i)
 {
+    // FIXME:
+    // The GCC version before 11 does not implement intrinsic function
```

Comment:
Why didn't you specify the conditional build with something like `if __GNUC__ <= 10`?